test/parallel/test-cli-node-options.js

```diff
@@ -24,6 +24,12 @@ expect('--throw-deprecation', 'B\n');
 expect('--zero-fill-buffers', 'B\n');
 expect('--v8-pool-size=10', 'B\n');
 expect('--trace-event-categories node', 'B\n');
+expect('--perf-basic-prof', 'B\n');
```

Comment:
This needs to be wrapped in `if (!common.isWindows)`

Reply:
I fixed it. Please re-run CI.